internal/wrappers/client.go

```diff
@@ -277,14 +280,25 @@ func HTTPRequestWithQueryParams(
 	return resp, nil
 }
 
+func getClaimsFromToken(tokenString string) (*jwt.Token, error) {
+	token, _, err := new(jwt.Parser).ParseUnverified(tokenString, jwt.MapClaims{})
```

Comment:
Is there a specific reason for parsing without validating the token signature?

Reply:
We are using HS256 algorithm and would need the same secret key used for encoding the token. Hence, using unverified version for now. We can re-visit this step in future.